styles.css

```diff
@@ -0,0 +1,53 @@
+html{
+    font-size: 12px;

```

Comment:
please remove this, because it will be overwritten by the font size declared in the body tag.